src/store/mutators.ts

```diff
@@ -161,6 +161,31 @@ export const updateAction = (nodes: RenderNodeMap, nodeUUID: string, action: Any
     });
 };
 
+export const spliceInAction = (
```

Comment:
feel like this closer to `convertToAction(..)`

Reply:
It's merging it into the `RenderNodeMap` as well though. Should this method only be concerned w/ converting the node to an action?